code.js

```diff
@@ -116,29 +107,22 @@ class StyleManager {
         return (style != undefined) ? style.id : null;
     }
 }
-class StyleKeyword {
-    constructor(name, mode) {
-        this.name = name;
-        this.mode = mode;
-    }
-    switch() {
-        if (!this.isModeKeyword) {
-            return this.name;
-        }
-        if (this.capitalized(this.name)) {
-            return this.capitalize(this.mode);
+class Replacer {
+    static replace(name, to) {
+        const keywords = Object.keys(Mode).map((key) => Mode[key]);
+        for (let from of keywords) {
+            if (name.match(from)) {
```

Comment:
Find and replace once with "light", "elevated dark" or "dark". Also L118 is same but it replaces with capitalized words.